pipelines/jobs/configurations/jdk21u_pipeline_config.groovy

```diff
@@ -166,6 +166,9 @@ class Config21 {
         riscv64Linux      :  [
                 os                  : 'linux',
                 arch                : 'riscv64',
+                // crossCompile        : 'dockerhost-rise-ubuntu2204-aarch64-1',
```

Comment:
@sxa can we actually enable this one as well?

Reply:
Since we're using 21 to bootstrap 21 on RISC-V then yeah as soon as we've got a suitable boot JDK that can be used then we can enable this.
On my "static" containerized build environemnts on my machines I've manually installed the [21.0.1+12.1 JDK](https://api.adoptium.net/v3/binary/version/jdk-21.0.1+12.1-ea-beta/linux/riscv64/jdk/hotspot/normal/adoptium) and symlinked to `/usr/lib/jvm/jdk-21` - we should set up the playbooks to do that.